pipelines/build/common/build_base_file.groovy

```diff
@@ -132,27 +133,56 @@ class Builder implements Serializable {
         return []
     }
 
+    def dockerOverride(Map<String, ?> configuration, String variant) {
+        Boolean overrideDocker = false
+        if (dockerExcludes == {}) {
+            return overrideDocker 
+        }
+
+        String stringArch = configuration.arch as String
+        String stringOs = configuration.os as String
+        String estimatedKey = stringArch + stringOs.capitalize()
+
+        if (configuration.containsKey("additionalFileNameTag")) {
```

Comment:
Possibly brittle?  Are there other suffixes that might apply when additionalFileNameTag is detected?

Reply:
Not that I know of. This code is also used to [form node labels](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/build/common/build_base_file.groovy#L176) that helps jenkins pick which machines should run which builds so if this is brittle, that makes the linked code brittle too